### PR TITLE
Reworked expanded pages, expanded meta text pages, and added page meta.

### LINKED ISSUE #194: Expand the meta text views (front-end PP/TOS view)
They're just too small.

### PR DESCRIPTION
**Pull request by Litarvan**
### Description

 - Reworked the "expanded" system by creating an "ExpandedView" component
 - Meta text pages are now expanded to improve the readability of the PDF
 - Added clearer and translated messages for the network error and rate limit error
 - Added meta description, image alts, and robots.txt
 - Some refractor and formatting changes were done too

#### Related issue(s)

Fixes #194 

### TODO

* [x] Update `CHANGELOG.md`

### CODE REVIEW

**Comment by Litarvan** [comment]
Fixed, fixed, and fixed :⁾

File: web/src/i18n/en.js [hunk 1]
```diff
@@ -83,16 +83,25 @@ export default {
...
+            about: 'About',
+            'not-found': 'Page not found'
         }
     },
 
     error: {
         title: 'Error',
-        retry: 'Retry'
+        retry: 'Retry',
+        network: 'Unable to reach the server, are we offline ? The server may be under maintenance.',
```
Comment (utybo) [inline web/src/i18n/en.js:103]:
```suggestion
        network: 'Unable to reach the server, please check your Internet connection. The server may also be under maintenance.',
```

File: web/src/i18n/fr.js [hunk 2]
```diff
@@ -83,16 +83,25 @@ export default {
...
+            'not-found': 'Page introuvable'
         }
     },
 
     error: {
         title: 'Erreur',
-        retry: 'Rééssayer'
+        retry: 'Rééssayer',
+        network: 'Impossible de contacter le serveur, veuillez vérifiez votre connexion. Le serveur est peut-être en maintenance.',
+        rateLimit: 'Vous avez été bloqués par le filtre anti-spam, veuillez réessayer dans quelques minutes.'
```
Comment (utybo) [inline web/src/i18n/fr.js:104]:
```suggestion
        retry: 'Réessayer',
        network: 'Impossible de contacter le serveur, veuillez vérifier votre connexion. Le serveur est peut-être en maintenance.',
        rateLimit: 'Vous avez été bloqué par le filtre anti-spam, veuillez réessayer dans quelques minutes.'
```

:voltaired:

File: web/src/views/Home.vue [hunk 3]
```diff
@@ -81,7 +81,7 @@
     }
 
     #discord {
-        $color: #7289DA;
+        $color: #687BBD;
```
Comment (utybo) [inline web/src/views/Home.vue:84]:
```suggestion
        $color: #7289DA;
```

The color in this PR is not Discord's official purple color. If a darker color is needed, we should use one of their official dark colors. 

image

Source: https://discord.com/new/branding

File: web/src/i18n/fr.js (hunk 2 above)
Comment (Litarvan) [reply web/src/i18n/fr.js]: J'ai fait ça dans le bus à 5h du mat pataper :upside_down_face:

File: web/src/views/Home.vue (hunk 3 above)
Comment (Litarvan) [reply web/src/views/Home.vue:84]: I know, but it was detected by validators such as web.dev as too bright for the text color, so it was the only way to achieve it right.

**Comment by Litarvan** [comment]
> the transition feels longer when switching between two expanded panes
I know about that, but this really can't be fixed without hard work that would kinda ruin the code base too.

**Pull request merged**